.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 .idea/
 dist/
 package-lock.json
+.env
```

Comment:
@IgorChaikin Good, but could you also add `.env.sample` file, which contains names of required environment variables (probably with empty or wrong values), and which is not included to `.gitingore`

<img width="291" alt="image" src="https://user-images.githubusercontent.com/12561302/127843399-03caff47-eb3f-4ba0-b87f-6727e9b248fe.png">